bin/animation_assets/manifest_builder.rb

```diff
@@ -6,12 +6,14 @@
 require_relative '../../lib/cdo/cdo_cli'
 require_relative '../../lib/cdo/png_utils'
 require_relative './constants'
+require_relative
```

Comment:
we aren't requiring a file here -- what does this do?

Reply:
Ahh whoops, that snuck in from the investigation work I'm doing to try and figure out the error this is giving.